qiskit/providers/ibmq/api_v2/rest/version_finder.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2018, 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Version finder for the IBM Q Api version 2."""
+
+from json import JSONDecodeError
+
+from .base import RestAdapterBase
+
+
+class VersionFinder(RestAdapterBase):
+    """Rest adapter for the version finder."""
+
+    URL_MAP = {
+        'version': '/version'
+    }
+
+    def version_info(self):
```

Comment:
Can you rename this method to `version()`? The (yet unwritten) convention for the `rest/` modules is that the name of the method is the same as the key in `URL_MAP` dict, for facilitating transitioning to something a bit more robust/automated in the future.

Reply:
Unresolving - can you rename the method?